.circleci/config.yml

```diff
@@ -3,12 +3,23 @@ version: 2.1
 executors:
   default:
     docker:
-    - image: circleci/golang:1.12
+      - image: ubuntu:18.04
     environment:
-      GORELEASER_VERSION: v0.108.0
       AWSCLI_VERSION: 1.16.122
 
 commands:
+  install_ansible:
+    description: "Install Ansible"
+    steps:
+      - run: 
+          name: Install Ansible & Dependencies
+          command: |
+            apt-get update
+            apt-get install -y software-properties-common python-apt python3-pip libssl-dev openssh-server
+            update-alternatives --install /usr/bin/python python /usr/bin/python3.6 2
+            update-alternatives  --set python /usr/bin/python3.6
+            pip3 install ansible
```

Comment:
These dependencies could go in the executable docker image (like https://github.com/cerebrotech/build-images/tree/master/domino-builder)

Reply:
Sure! https://github.com/cerebrotech/build-images/pull/16